src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.EcKey.cs

```diff
@@ -38,7 +38,7 @@ internal static int EcKeyGetSize(SafeEcKeyHandle key)
             throw Interop.Crypto.CreateOpenSslCryptographicException();
         }
 
-        [DllImport(Libraries.CryptoNative)]
+        [DllImport(Libraries.CryptoNative, EntryPoint = "CryptoNative_EcKeyGetCurveName")]
```

Comment:
Instead of m.s.c.openssl.dll it should have been s.s.c.opensll.dll (system.security.cryptography.opensll.dll)